RavenDB-12610

ppekrol opened the pull request:
- WaitForExecutionOnRelevantNodes needs to wait for execution on all nodes, not the first one

arekpalinski commented on src/Raven.Server/Web/System/AdminDatabasesHandler.cs:
```diff
@@ -428,44 +429,67 @@ private async Task WaitForExecutionOnRelevantNodes(JsonOperationContext context,
+                            throw new TimeoutException(
+                                $"Waited too long for the raft command (number {index}) to be executed on any of the relevant nodes to this command.", oce);
+                        }
+                        catch (RavenException e) when (e.InnerException is HttpRequestException)
```
What if two of them fail with this error, won't we get aggregate exception then?

ppekrol replied:
Fixed

ppekrol committed:
RavenDB-12610
- WaitForExecutionOnRelevantNodes needs to wait for execution on all nodes, not the first one
